example/docker-compose/local-distributed/docker-compose.yaml

```diff
@@ -0,0 +1,122 @@
+version: "3"
+services:
+
+  distributor:
+    image: tempo:latest
+    command: "-target=distributor -config.file=/etc/tempo.yaml"
+    volumes:
+      - ./tempo-distributed.yaml:/etc/tempo.yaml
+      - ./tempo-data:/tmp/tempo
+    ports:
+      - "14268"  # jaeger ingest
+      - "3200"   # tempo
+      - "7946"
+      - "55680"  # otlp grpc
+      - "55681"  # otlp http
+
+  ingester-0:
+    image: tempo:latest
+    command: "-target=ingester -config.file=/etc/tempo.yaml"
+    restart: always
+    volumes:
+      - ./tempo-distributed.yaml:/etc/tempo.yaml
+      - ./tempo-data0:/tmp/tempo
+    ports:
+      - "14268"  # jaeger ingest
+      - "3200"   # tempo
+      - "7946"
+    depends_on:
+      - distributor
+
+  ingester-1:
+    image: tempo:latest
+    command: "-target=ingester -config.file=/etc/tempo.yaml"
+    restart: always
+    volumes:
+      - ./tempo-distributed.yaml:/etc/tempo.yaml
+      - ./tempo-data1:/tmp/tempo
+    ports:
+      - "14268"  # jaeger ingest
+      - "3200"   # tempo
+      - "7946"
+    depends_on:
+      - distributor
+
+  ingester-2:
+    image: tempo:latest
+    command: "-target=ingester -config.file=/etc/tempo.yaml"
+    restart: always
+    volumes:
+      - ./tempo-distributed.yaml:/etc/tempo.yaml
+      - ./tempo-data2:/tmp/tempo
+    ports:
+      - "14268"  # jaeger ingest
+      - "3200"   # tempo
+      - "7946"
+    depends_on:
+      - distributor
+
+  tempo:
+    image: tempo:latest
+    command: "-target=query-frontend -config.file=/etc/tempo.yaml"
+    volumes:
+      - ./tempo-distributed.yaml:/etc/tempo.yaml
+      - ./tempo-data:/tmp/tempo
+    ports:
+      - "14268"  # jaeger ingest
+      - "3200"   # tempo
+      - "7946"
+      - "55680"  # otlp grpc
+      - "55681"  # otlp http
+    depends_on:
+      - ingester-0
+      - ingester-1
+      - ingester-2
+
+  querier:
+    image: tempo:latest
+    command: "-target=querier -config.file=/etc/tempo.yaml"
+    volumes:
+      - ./tempo-distributed.yaml:/etc/tempo.yaml
+      - ./tempo-data:/tmp/tempo
```

Comment:
Can remove.

Reply:
Doesn't the querier need access to the storage to query blocks flushed by the ingesters? But I'm not even sure how the ingesters will flush their blocks in this setup 🤔  I have never run the local backend in distributed mode.
It might be easier to use a containerised backend like fake (gcs) or minio (s3).

Reply:
Ah that's right. We need this one.